components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -89,13 +89,108 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
+
+{{< figure src="/images/automate/create-environment-popup.png" alt="Create Environment Dialog Box">}}
+
+#### Search for an Environment
+
+Use the **Search environments** bar to find an existing environment from the list of environments.
+Entering the name of an environment in the search box returns environments matching your search criteria.
+
+{{< figure src="/images/automate/create-environment-button.png" alt="Create Environment Button">}}
+
+#### Details of an Environment
+
+To view the details of an environment, follow the steps mentioned below:
+
+- Select a specific environment.
+  - **Cookbook Constraints:** The selected bar contains the list of all the cookbook constraints with _Name_, their _Operator_, and _Version_ respectively. Select **Edit** to change the _Cookbook Constraints_.
+
+  {{< figure src="/images/automate/cookbook-constraints-in-environment.png" alt="Cookbook Constraints In Environment">}}
+
+  - **Attributes:** The attributes bar shows all the default and overridden attributes. Select **Expand All** or **Collapse All** to view or hide the attributes. Select **Edit** to change the _Default Attributes_ and _Override Attributes_.
+
+  {{< note >}} You can edit _Cookbook Constraints_ and the _Attributes_ only for environments with an existing _client_ for that _organization_.
+  {{< /note >}}
+
+  {{< figure src="/images/automate/attributes-in-environment.png" alt="Attributes In Environment">}}
+
+#### Delete an Environment
+
+The Chef Infra Server lets you delete environments one at a time. Select **Delete** from the more information (three dots) icon on the far right side of the environment listing that you want to delete, as shown below:
+
+{{< figure src="/images/automate/delete-an-environment.png" alt="Delete an Environment by selecting the more information icon on the right side of an environment list item">}}
+
 ### Data Bags
 
 [Data bags]({{< relref "data_bags" >}}) store global variables as JSON data. Data bags are indexed for searching and can be loaded by a cookbook or accessed during a search.
 
 ### Clients
 
-Chef Infra Clients provide secure API access to the Chef Infra Server.
+Chef Infra Clients provide secure API access to the Chef Infra Server. Chef Infra Server UI lets you:
+
+- Create a client.
+- Search for a specific client from a list of clients.
+- List out all the clients at one place.
+- Reset a client key.
+- Delete a client.
+
+#### Create a Client
+
+To create a new client, select **Create Client**.
+
+{{< figure src="/images/automate/create-client-button.png" alt="Create Client Button">}}
+
+Selecting the **Create Client** button opens a dialog box. The dialog box contains a section to add _Client Name_ and also an option to select whether the client is _Validation Client_ or not. Select **Create** .
```

Comment:
What's the purpose of a Validation Client? This should probably link to docs that explain this so people know if they should check this or not. I'm assuming it has to do with `validation_client_name` here: https://docs.chef.io/config_rb_client/#settings

```suggestion
The **Create Client** button opens a dialog box. Enter the _Client Name_ and select _Validation Client_ or not. Select **Create** .
```

Reply:
ok